packages/babel-plugin-named-asset-import/package.json

```diff
@@ -13,5 +13,16 @@
   ],
   "peerDependencies": {
     "@babel/core": "^7.1.0"
+  },
+  "devDependencies": {
+    "@babel/preset-env": "^7.1.0",
```

Comment:
Is this dependency and `regenerator-runtime` necessary?

Reply:
it is removed as not needed more